openedx/core/lib/xblock_serializer/utils.py

```diff
@@ -89,6 +94,38 @@ def collect_assets_from_text(text, course_id, include_content=False):
         yield info
 
 
+def get_python_lib_zip_if_using(olx: str, course_id: CourseKey) -> StaticFile | None:
+    """
+    When python_lib is in use, capa problems that contain python code should be assumed to depend on it.
+
+    Note: for any given problem that uses python, there is no way to tell if it
+    actually uses any imports from python_lib.zip because the imports could be
+    named anything. So we just have to assume that any python problems may be
+    using python_lib.zip
+    """
+    MATCH_STRINGS = (
+        '<script type="text/python">',
+        "<script type='text/python'>",
+        '<script type="loncapa/python">'
```

Comment:
```suggestion
        '<script type="loncapa/python">',
```

Python does not make things easy by allowing string concatenation this way.

It would be good to add all these cases to the tests. This way, we can catch if anybody removes one of these lines.